.github/workflows/deploy.yml

```diff
@@ -32,9 +35,19 @@ jobs:
       - name: Build plugin
         run: npm run build
 
-      - name: WordPress Plugin Deploy
+      - name: WordPress.org asset and readme update
+        uses: 10up/action-wordpress-plugin-asset-update@stable
+        if: github.ref_name == 'master'
+        env:
+          # Note: this action doesn't support BUILD_DIR so it pushes the raw readme.txt
+          IGNORE_OTHER_FILES: true
+          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+
+      - name: WordPress.org deploy
         id: deploy
         uses: 10up/action-wordpress-plugin-deploy@stable
+        if: startsWith( github.ref, 'refs/tags/' )
```

Comment:
This action derives the release VERSION from the tag name so it can't run anywhere else but on tag builds.